code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm

```diff
@@ -165,7 +171,7 @@
 	gain_text = "Uncanny man, walks alone in the valley between the worlds , I was able to call his aid."
 	desc = "You can now summon a Raw Prophet using eyes, a left arm, right arm and a pool of blood. Raw prophets have increased seeing range, as well as Xray. But are very fragile and weak."
 	cost = 1
-	required_atoms = list(/obj/item/organ/eyes,/obj/item/bodypart/l_arm,/obj/item/bodypart/r_arm,/obj/effect/decal/cleanable/blood)
```

Comment:
aw, it was nice being able to turn my gibbed sacrifices (whose heads I'd then butcher with a saw) into raw prophets with virtually no resource loss